deltachat-ffi/src/lib.rs

```diff
@@ -3246,7 +3246,7 @@ pub unsafe extern "C" fn dc_lot_get_text2(lot: *mut dc_lot_t) -> *mut libc::c_ch
     }
 
     let lot = &*lot;
-    strdup_opt(lot.get_text2())
+    lot.get_text2().strdup()
```

Comment:
there are only two invocations of the new code. I'd rather hope taht at some point we return a seralized rust-struct instead of using dc_lot, and then the strdup would go away completely.